### PR TITLE
Editor revision 2021 05 21

### PR DESCRIPTION
**Pull request by tschmidtb51**
- bring editor revision back in sync with master
- minor changes

### CODE REVIEW

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 1]
```diff
@@ -1706,6 +1706,23 @@ Examples:
...
+2. Each character which is not part of one of the following groups MUST be replaced by an underscore (`_`):
+   * Lower case ASCII letters (0x61 - 0x7A)
+   * digits (0x30 - 0x39)
+   * special characters: `+` (0x2B), `-` (0x2D), `_` (0x5F)
+3. The file extension `.json` MUST be appended.
+
+Examples:
+
+```
+    example_company_-_2019-yh3234.json
```
Comment (sthagen) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:1721]: Nit: We should order examples, I think, but we can do later (and ensure we do the same in the schema)

**Review by sthagen — APPROVED** [review]
LGTM.

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 1 above)
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md:1722]: This is now tracked as #264.

**Comment by tschmidtb51** [comment]
@sthagen: Thank you.

**Pull request merged**